package.json

```diff
@@ -40,15 +40,24 @@
     "@mantine/next": "^5.10.4",
     "@mantine/notifications": "^5.10.4",
     "@mantine/spotlight": "^5.10.4",
+    "@nivo/bump": "^0.80.0",
+    "@nivo/core": "^0.80.0",
+    "@nivo/line": "^0.80.0",
     "@tabler/icons": "^1.106.0",
+    "chart.js": "^4.2.1",
     "dayjs": "^1.11.7",
     "firebase": "^9.17.1",
     "json-2-csv": "^3.19.0",
+    "inherits": "^2.0.4",
```

Comment:
Do we need this? I think we can remove it

Reply:
yes, I also removed it in my package.json now.